featuremanagement/_models/_targeting_context.py

```diff
@@ -0,0 +1,19 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+from typing import NamedTuple, List
+
+
+class TargetingContext(NamedTuple):
+    """
+    Represents the context for targeting a feature flag.
```

Comment:
Would you mind standardizing the punctuation in the comments throughout the repo? It would be great if they could all end with a period or not.